iib/workers/tasks/utils.py

```diff
@@ -326,7 +319,12 @@ def skopeo_inspect(*args, return_json=True):
     cmd = ['skopeo', '--command-timeout', skopeo_timeout, 'inspect'] + list(args)
     output = run_cmd(cmd, exc_msg=exc_msg)
     if return_json:
-        return json.loads(output)
+        output_json = json.loads(output)
+        if output_json.get('mediaType') == None:
```

Comment:
Is `mediaType` field in every output of `skopeo inspect`?

TIP: `None` type is better compare using `is` operator `if output_json.get('mediaType') is None`.

Reply:
I guess `mediaType` does not have to be present in output of `skopeo inspect`.
Therefore the above code will not work properly, because get() will always return `None` when `mediaType` is not in output.

Reply:
And other question is: should we focus only on case when we return json? Shouldn't the retry be always when `mediaType` is `None`?